### PR TITLE
Adding Normalize.scss, namespacing home-services and service components.

### PR DESCRIPTION
**Pull request by astolfivincent**

### CODE REVIEW

File: components/home-service/home-service.scss [hunk 1]
```diff
@@ -6,9 +6,77 @@
...
+}
+
+.home-service__service-title > a {
+  color: $home-service-white;
+  text-decoration: none;
+  font-family: sans-serif;
+}
+
+
+.BlogTeaser__time,
```
Comment (rcaracaus) [inline components/home-service/home-service.scss]: these things don't need to be in here

Comment (rcaracaus) [inline components/home-service/home-service.scss]: only home-service stuff

File: components/home-service/home-service.css [hunk 2]
```diff
@@ -1,1153 +0,0 @@
-html {
```
Comment (rcaracaus) [inline components/home-service/home-service.css]: still need to add normalize stuff in here

Comment (rcaracaus) [inline components/home-service/home-service.css]: In scss/base

**Comment by astolfivincent** [comment]
Is it cool if I merge this now? The next issue I'm working on is dealing with some of the same files.

**Comment by rcaracaus** [comment]
I will always merge, you can branch your next issue off this issue if you need stuff in this one.

File: components/header/header.scss [hunk 3]
```diff
@@ -46,6 +46,19 @@ $dark-grey: #222;
   display: inline-block;
 }
 
+.header__logo a {
```
Comment (rcaracaus) [inline components/header/header.scss]: please add a class directly to the a tag, we should never be specifying raw tags in components.

**Comment by astolfivincent** [comment]
Oh duh, will do!

File: components/header/header.scss [hunk 4]
```diff
@@ -46,6 +46,19 @@ $dark-grey: #222;
...
+  display: block;
+  overflow: hidden;
+  width: 195px;
+  height: 60px;
+  margin: 7px auto 0px auto;
+  transition: width .5s;
+}
+
+.header__logo a:hover {
```
Comment (rcaracaus) [inline components/header/header.scss]: this should be &:hover inside the selector above

File: scss/styles.scss [hunk 5]
```diff
@@ -1,5 +1,5 @@
 @import '../node_modules/breakpoint-sass/stylesheets/breakpoint';
-
+@import "../node_modules/normalize.scss/normalize.scss";
```
Comment (rcaracaus) [inline scss/styles.scss]: this will force us to override normalize when I would prefer to fork it, good thinking to npm install it though.

Comment (rcaracaus) [inline scss/styles.scss]: But unfortunately this one goes against common logic and we should copy paste it into scss/base

**Pull request merged**